tools/check_envoy_includes.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+# Ensure that includes are referenced via external/envoy
+
+import os, sys
+from pathlib import Path
+
+def get_inspection_targets_from_dir(dir):
+    result = []
```

Comment:
Does this match the style in https://github.com/envoyproxy/nighthawk/blob/master/test/integration/integration_test.py? Ideally same style, check/fix_format applies.

Reply:
I ran it through fix fomat to resolve, ptal?